src/runtime/entitlements-manager.js

```diff
@@ -456,6 +461,23 @@ export class EntitlementsManager {
     return this.responsePromise_.then(() => Promise.resolve(this.article_));
   }
 
+  /**
+   * The experiment flags that are returned by the article endpoint should be accessible from here.
+   * @returns {Promise<Array<string>>}
+   */
+  getExperimentConfigFlags() {
+    return this.getArticle().then((article) => {
+      const expConfig = article['experimentConfig'];
+      if (expConfig != null) {
+        const expFlags = expConfig['experimentFlags'];
+        if (expFlags != null) {
+          return expFlags;
+        }
+        return [];
```

Comment:
Could you add a test that covers this line?
https://app.codecov.io/gh/subscriptions-project/swg-js/compare/2128/diff

Reply:
Yes! Done.